### ISSUE TITLE
Access value/message after start-up 

### ISSUE THREAD

**Issue by venzen** [henriquehbr/nvim-startup.lua#8]
I 'm using the nvim-alpha plugin as a dashboard. Unfortunately, the startup-time message is being erased from the command/message window/ It only flashes briefly.

Is it possible to access the startup time value (in milisecs) or the startup-time message from within my nvim session?

**Comment by venzen** [comment]
That would be great! And it exposes startup time to developers of other plugins such as statusline, lualine, etc...

**Comment by henriquehbr** [comment]
There you go, at the moment, this change is at the `dev` branch, i tested it locally, and it's working fine for me

Even though it's something simple and trivial, can you test it as well in order to ensure everything is working as intended?

If so, we can merge it to `main` and call it a day

> Remember to set the `branch` option to `dev` on the `nvim-startup` entry of your plugin manager so you get the absolute latest version containing this change

**Comment by venzen** [comment]
Thanks for this - tested and it works as intended. 
Used the option `branch = 'dev'` in Packer and was able to access `startup _time`.

Question: is the variable `startup_time` accessible via the vim.api or anywhere in the vim state? Can you perhaps suggest a way to expose `startup_time` to be accessible from another plugin's config files?

**Comment by henriquehbr** [comment]
Thanks a lot for your valuable contribution @venzen!

I saw your comment on the commit, and just merged this to `main` and released as [`v0.6.0`](https://github.com/henriquehbr/nvim-startup.lua/releases/tag/v0.6.0)

---

Currently, `startup_time` is not accessible via the `vim.api`, some of the solutions that i could think to have access to the startup time variable would be:

- Create a separated module for `nvim-startup`, run it's setup function as soon as possible (before any other plugins) on your config, and export the `startup_time` variable so it can be `require`'d on other files
- Storing it on a global variable (i'm not sure whether this would be an ideal solution, because of the "globals are bad" dogma)

I don't know about any plugins that make use of `vim.api` to store state data, but feel free to share any that you might know of

**Comment by venzen** [comment]
In case it helps anyone, here's how I implemented this.

As @henriquehbr describes above, startup time can be accessed either via requiring `nvim-startup` locally in a lua script, or the startup time value can be accessed (later) by declaring a global var when setting up `nvim-startup` the first time. 

*Global var*

Use a long compound name to avoid collision with other global variables, either now or in the future:
```lua
local start_ok, startuptime = pcall(require, "nvim-startup")
if not start_ok then
  return
end

Startup_time_from_plugin = startuptime.setup()
```
This global variable can then be accessed from another script (or cmd window) 
```lua
local starttime = vim.inspect(Startup_time_from_plugin)
```

*Local only*
```lua
local nvim_startup = require 'nvim-startup'
local stt = nvim_startup.setup {}
print(stt)
```
The benefit of using `require` is that it does not create a second instance of the called script, but instead uses a cached instance, so you get the actual startup time calculation when it was invoked the first time.

I went with the global variable because it is easier and only private to my environment anyway.
